src/pip/_internal/wheel.py

```diff
@@ -852,19 +854,22 @@ def _build_one_pep517(self, req, tempd, python_tag=None):
                 tempd,
                 metadata_directory=req.metadata_directory
             )
+            wheel_path = os.path.join(tempd, wheelname)
             if python_tag:
                 # General PEP 517 backends don't necessarily support
                 # a "--python-tag" option, so we rename the wheel
                 # file directly.
                 newname = replace_python_tag(wheelname, python_tag)
-                os.rename(
-                    os.path.join(tempd, wheelname),
-                    os.path.join(tempd, newname)
-                )
-            return True
+                new_path = os.path.join(tempd, newname)
+                os.rename(wheel_path, new_path)
+
+                # This makes the return below easier.
+                wheel_path = new_path
         except Exception:
             logger.error('Failed building wheel for %s', req.name)
-            return False
+            return None
+        else:
+            return wheel_path
 
     def _build_one_legacy(self, req, tempd, python_tag=None):
```

Comment:
I would document the return value. Is it a path or a filename? (and similar for the other method)

Reply:
Done!